lib/list.gd

```diff
@@ -170,7 +170,7 @@ InstallTrueMethod(HasLength,IsPlistRep);
 ##  <#/GAPDoc>
 ##
 DeclareOperationKernel( "IsBound[]",
-    [ IsList, IsObject ],
+    [ IsList, IS_INT ],
```

Comment:
IIRC `IsPosInt` is not available at that point, and we used `IS_INT` before, too -- but I need to check up on that